plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -2989,16 +2989,18 @@ public void testVoidTransform()
         assertQuery("SELECT * FROM test_void_transform", values);
 
         assertQuery("SELECT COUNT(*) FROM \"test_void_transform$partitions\"", "SELECT 1");
-        if (format != AVRO) {
-            assertQuery(
-                    "SELECT partition.d_null, record_count, file_count, data.d.min, data.d.max, data.d.null_count, data.d.nan_count, data.b.min, data.b.max, data.b.null_count, data.b.nan_count FROM \"test_void_transform$partitions\"",
-                    "VALUES (NULL, 7, 1, 'Warsaw', 'mommy', 2, NULL, 1, 7, 0, NULL)");
-        }
-        else {
-            assertQuery(
-                    "SELECT partition.d_null, record_count, file_count, data.d.min, data.d.max, data.d.null_count, data.d.nan_count, data.b.min, data.b.max, data.b.null_count, data.b.nan_count FROM \"test_void_transform$partitions\"",
-                    "VALUES (NULL, 7, 1, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL)");
-        }
+
+        // TODO: After https://github.com/apache/iceberg/pull/3059 a partition spec with void transform is considered unpartitioned
```

Comment:
@findepi any suggestions to what we want to do in this test here due to the implications from https://github.com/apache/iceberg/pull/3059?

Reply:
for now just adjust assertion so that is passes, and then we can see what's the effect of these changes.
maybe it's just 🤷‍♂️ , or maybe it's a problem. to be seen